controllers/places.js

```diff
@@ -252,3 +269,4 @@ module.exports.createPlace = createPlaces;
 module.exports.getPlace = getPlace;
 module.exports.updatePlace = updatePlace;
 module.exports.getOrCreatePlace = getOrCreatePlace;
+module.exports.minify = minify;
```

Comment:
`minify` is used outside this module so it shouldn't have the underscore. Soon we can extract it into your library but for now this is a decent place to export it from.

IMO we should stop using the underscore prefix convention. My preferred way is to break each module up so it's small enough to unit test without having to expose the guts of it. If that's not practical then the next best way is to export more function when the TEST_ENV is set. That way you can't accidentally use one in production code.